app/src/main/java/ch/epfl/skysync/components/AvailabilityCalendar.kt

```diff
@@ -4,32 +4,36 @@ import androidx.compose.foundation.background
 import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.material.Button
-import androidx.compose.material.Text
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
 import androidx.compose.runtime.*
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import ch.epfl.skysync.models.calendar.AvailabilityStatus
 import ch.epfl.skysync.models.calendar.TimeSlot
+import ch.epfl.skysync.ui.theme.*
 import java.time.LocalDate
 
 // Define custom colors to represent availability status
 // These colors are used to indicate availability status in the UI
 
 // Custom color for indicating availability status as "OK" (e.g., available)
-val customGreen = Color(android.graphics.Color.parseColor("#aaee7b"))
+val customGreen = Color(0xffaaee7b)
```

Comment:
What is the difference in usage between this one and the one added in Color?

Reply:
It the the same color but the second solution is shorter